web/src/features/moreCast2/pages/MoreCast2Page.tsx

```diff
@@ -80,6 +87,10 @@ const useStyles = makeStyles(theme => ({
 const DEFAULT_MODEL_TYPE_KEY = 'defaultModelType'
 const DEFAULT_FIRE_CENTER_KEY = 'preferredMoreCast2FireCenter'
 
+const FORECAST_ERROR_MESSAGE = 'The forecast was not saved; an unexpected error occurred'
+const FORECAST_SAVED_MESSAGE = 'Forecast was successfully saved'
+const FORECAST_WARN_MESSAGE = 'A forecast cannot contain N/A values.'
```

Comment:
Pedantry Level 1000:

Only the warning message has a . at the end. A content designer would likely argue that our punctuation should be consistent.

Reply:
(You would be entirely justified in ignoring this. I would probably ignore myself in your situation.)

Reply:
Good eye and good idea. I'll fix it.